src/click_handler.coffee

```diff
@@ -18,12 +18,18 @@ define [
       if !$target.is(@clickBindSelector)
         $target = $target.parent()
 
+      targetType = ($target, @finder) ->
+        if @finder.item($target) == 'phone_number_link'
+          'lead_submission'
+        else
+          'click'
```

Comment:
Hmm good question.  We were able to create a mapping for this in Tealium, which is we don't have available here.  I'm ok with this implementation for now, but it will definitely need to get refactored later.